src/emulator/emulatorLogger.ts

```diff
@@ -121,25 +121,19 @@ export class EmulatorLogger {
       case "googleapis-network-access":
         EmulatorLogger.log(
           "WARN",
-          `Google API requested!\n   - URL: "${
-            systemLog.data.href
-          }"\n   - Be careful, this may be a production service.`
+          `Google API requested!\n   - URL: "${systemLog.data.href}"\n   - Be careful, this may be a production service.`
```

Comment:
Did this change our max line length? Or is there a special allowance for long strings.  Fwiw I like this change just wondering why it happened.

Reply:
I think prettier used to let you have longer strings but when template strings came along they made the decision to break within the `${}` blocks. Evidently that's turned around.

Personally (don't change this right now because it's fine), I would rather see `EmulatorLogger.log` functions for each line rather than injecting `\n` into the strings. It's much harder to reason about when logs aren't on one line (maybe this shouldn't have newlines at all?)